### PR TITLE
[9.0][Port] mig partner changeset

### PR DESCRIPTION
**Pull request by leemannd**
WIP
As https://github.com/OCA/partner-contact/pull/213 is not going forward, there is a proposition to do it here.
[EDIT]
- Code from 8.0 taken, with the newest commits
- Update tests
- Updates changes made in other's PR
- Fix menu

### CODE REVIEW

**Comment by leemannd** [comment]
@pedrobaeza @guewen @damdam-s It is not anymore in WIP state

File: partner_changeset/security/security.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+  <data>
```
Comment (leemannd) [inline partner_changeset/security/security.xml:3]: Not sure about this one. Is it possible with `<odoo>` to have a part of the records with noupdate and a part without?

**Review by jgrandguillaume — APPROVED** [review]
Hi,


Functional test done, works great ! Thanks for the port :+1: 

Regards,

Joël

**Review by damdam-s — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I know I'm always asking the same, but can you squash all that useless commits from the same person?

**Comment by leemannd** [comment]
@pedrobaeza I usually don't feel comfortable with modifying the git history of other people, but I'll do it in the beginning when porting a module.
Done.

File: partner_changeset/README.rst [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  .. image:: partner_changeset/static/src/img/rules.png
+
+* Changeset waiting for validation
+
+  .. image:: partner_changeset/static/src/img/changeset.png
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/134/8.0
```
Comment (pedrobaeza) [inline partner_changeset/README.rst]: Change this to 9.0

File: partner_changeset/README.rst [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/partner-contact/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+partner-contact/issues/new?body=module:%20
+partner_changeset%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (pedrobaeza) [inline partner_changeset/README.rst]: And remove this part according last template

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Little changes that you can squash with your main PR

**Comment by leemannd** [comment]
@pedrobaeza Corrected

**Comment by pedrobaeza** [comment]
Travis is failing

**Comment by leemannd** [comment]
@pedrobaeza error in tests linked to the fact that cache is not updated during the same transaction. It should be better now.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**